WooCommerce/Classes/ViewRelated/Products/View Models/ProductsTabProductViewModel.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import Yosemite
+
+/// Converts the input product model to properties ready to be shown on `ProductsTabProductTableViewCell`.
+struct ProductsTabProductViewModel {
+    let imageUrl: String?
+    let name: String
+    let detailsAttributedString: NSAttributedString
+
+    init(product: Product) {
+        imageUrl = product.images.first?.src
+        name = product.name
+        detailsAttributedString = product.createDetailsAttributedString()
+    }
+}
+
+private extension Product {
+    func createDetailsAttributedString() -> NSAttributedString {
+        let statusText = createStatusText()
+        let stockText = createStockText()
+        let variationsText = createVariationsText()
+
+        let detailsText = [statusText, stockText, variationsText]
+            .compactMap({ $0 })
+            .joined(separator: " • ")
```

Comment:
Should this be a unicode character? Or do all languages have this dot?

Reply:
• is a unicode character: `U+2022 • bullet` as in the [Wikipedia](https://en.wikipedia.org/wiki/Bullet_(typography))